src/theory/quantifiers/sygus/sygus_grammar_cons.h

```diff
@@ -186,6 +187,36 @@ class CegGrammarConstructor
       Node n,
       std::map<TypeNode, std::unordered_set<Node, NodeHashFunction>>& consts);
   //---------------- grammar construction
+  /** A class for generating sygus datatypes */
+  class SygusDatatypeGenerator
+  {
+   public:
+    SygusDatatypeGenerator(const std::string& name);
+    ~SygusDatatypeGenerator() {}
+    /** Possibly add a constructor to d_sdt, based on the above criteria. */
```

Comment:
above where?

Reply:
Oops, got rearranged. Will fix the comment.